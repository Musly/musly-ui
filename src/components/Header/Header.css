.header {
  position: fixed;
  top: 0;
  right: 0;
  left: var(--msly-nav-width);
  height: var(--msly-header-height);
  display: flex;
  min-width: calc(var(--msly-min-width) - var(--msly-nav-width));
  z-index: 10;
  align-items: center;

  @apply bg-gray-50 shadow-header;
}
