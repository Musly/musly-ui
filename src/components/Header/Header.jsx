import React from 'react';
import { UserMenu } from '../../features/User/UserMenu';
import { Search } from './Search';
import './Header.css';

export function Header () {
  return (
    <header className="header" role="banner">
      <Search />
      <div className="flex-grow" />
      <UserMenu />
    </header>
  );
}
