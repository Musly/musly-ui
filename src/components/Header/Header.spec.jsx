import React from 'react';
import { shallow } from 'enzyme';
import { Header } from './Header';

describe('<Header />', () => {
  it('should render as expected', () => {
    const result = shallow(<Header />);
    expect(result.find('Search')).toExist();
    expect(result.find('UserMenu')).toExist();
    expect(result).toMatchSnapshot();
  });
});
