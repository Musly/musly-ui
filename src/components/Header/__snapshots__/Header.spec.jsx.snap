// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render as expected 1`] = `
<header
  className="header"
  role="banner"
>
  <Search />
  <div
    className="flex-grow"
  />
  <UserMenu />
</header>
`;
